boards/arm/nrf52_pca20020/doc/nrf52_pca20020.rst

```diff
@@ -0,0 +1,354 @@
+| nRF52832 | VDD_nRF |
++----------+---------+
+| SX1509B  | VDD     |
++----------+---------+
+| LPS22HB  | VDD     |
++----------+---------+
+| HTS221   | VDD     |
++----------+---------+
+| CCS811   | VDD_CCS |
++----------+---------+
+
+Sensors
+-------
+
++----------+---------------------------------+-------+-------------+------------+
+| Device   | Function                        | Bus   | I2C Address | Power Rail |
++==========+=================================+=======+=============+============+
+| LPS22HB  | Pressure and Temperature sensor | I2C_0 | 0x5c        | VDD        |
++----------+---------------------------------+-------+-------------+------------+
+| HTS221   | Humidity and Temperature sensor | I2C_0 | 0x5f        | VDD        |
++----------+---------------------------------+-------+-------------+------------+
+| CCS811   | Gas sensor                      | I2C_0 | 0x5a        | VDD_CCS    |
++----------+---------------------------------+-------+-------------+------------+
+
+Misc. Device Pins
+-----------------
+
+SX1509B
+~~~~~~~
+
++---------------+-------------------+
+| Device Signal | SOC Pin           |
++===============+===================+
+| SX_OSCIO      | P0.05             |
++---------------+-------------------+
+| SX_RESET      | P0.16             |
++---------------+-------------------+
+
+LPS22HB
+~~~~~~~
+
++---------------+-------------------+
+| Sensor Signal | SOC Pin           |
++===============+===================+
+| LPS_INT       | P0.23             |
++---------------+-------------------+
+
+HTS221
+~~~~~~
+
++---------------+-------------------+
+| Sensor Signal | SOC Pin           |
++===============+===================+
+| HTS_INT       | P0.24             |
++---------------+-------------------+
+
+CCS811
+~~~~~~
+
++---------------+-------------------+
+| Sensor Signal | GPIO Expander Pin |
++===============+===================+
+| CCS_RESET     | 11                |
++---------------+-------------------+
+| CCS_WAKE      | 12                |
++---------------+-------------------+
+
+Programming and Debugging
+*************************
+
+Flashing
+========
+
+Flashing Zephyr onto Thingy:52 requires an external J-Link programmer. The
+programmer is attached to the P9 programming header.
+
+
+Debugging
+=========
+
+Thingy:52 does not have an on-board J-Link debug IC like some other nRF5
+development boards, however, instructions from the :ref:`nordic_segger` page
+also apply to this board, with the additional step of connecting an external
+debugger. A development board with a Debug out connector like the
+:ref:`nrf52_pca10040` can be used as a debugger with Thingy:52.
+
+Testing board features
+**********************
+
+The green lightwell LED can be tested with the blinky example.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/basic/blinky
+   :board: nrf52_pca20020
+   :goals: build flash
+
+
+
+Also the temperature and humidity sensor can be tested with the following
```

Comment:
change ``following sample`` to `` :ref:`hts221` sample`` (links to the sample's documentation)